ynh-dev

```diff
@@ -306,6 +308,52 @@ elif [ "$1" = "use-git" ]; then
     done
 
 
+elif [ "$1" = "test" ]; then
+    check_yunohost_vm
+    VERSION=$2
+
+    for i in ${!packages[@]}; do
+        case ${packages[i]} in
+            yunohost)
+                # Pytest and tests dependencies
+                if ! type "pytest" > /dev/null; then
+                    echo "======================="
+                    echo "> Installing pytest ..."
+                    echo "======================="
+                    apt-get install python-pip
+                    pip2 install pytest
+                fi
+                PIP_DEPENDENCIES="pytest-mock requests-mock"
+                for DEP in $PIP_DEPENDENCIES
+                do
+                    if [ -z "pip show $DEP" ]; then
```

Comment:
I tested [Refactor applist management tests](https://github.com/YunoHost/yunohost/pull/160/files#diff-4be4fde016c663bcc1ab209bf3098ff0R4) and needed to install `requests_mock` python package manually.
I correct it.